Comp2024/src/main/java/frc/robot/subsystems/HID.java

```diff
@@ -23,12 +23,16 @@ public class HID extends SubsystemBase
   // Member objects
   private GenericHID m_driver;
   private GenericHID m_operator;
+  private Timer      m_timerDriver   = new Timer( );
+  private Timer      m_timerOperator = new Timer( );
+  private Boolean    m_driverRumbleOn;
+  private Boolean    m_operatorRumbleOn;
 
   /****************************************************************************
    * 
    * Constructor
    */
-  public HID(GenericHID driver, GenericHID operator)
+  public HID(GenericHID driver, GenericHID operator, Boolean driverRumbleOn, Boolean operatorRumbleOn)

```

Comment:
We don't need to pass in an initial rumble state when we created the HID subsystem. Those last two parameters should be deleted.

When declaring m_driverRumbleOn as class member variables (and operator), it's a good practice to initialize them to false.

private Boolean m_driverRumbleOn = false;

Reply:
fixed